### PR TITLE
Add canceled quantity to line item serializer. Clean up specs

### PR DESCRIPTION
**Pull request by philbirt**
cc @jordan-brough

### CODE REVIEW

File: spec/serializers/spree/hub/line_item_serializer_spec.rb [hunk 1]
```diff
@@ -15,6 +15,9 @@ module Wombat
         expect(JSON.parse(serialized_line_item)["price"].class).to eql Float
       end
 
+      it "serializes the canceled quantity" do
```
Comment (jordan-brough) [inline spec/serializers/spree/hub/line_item_serializer_spec.rb]: should we also have a spec with a non-zero cancellation quantity?

**Comment by jordan-brough** [comment]
:+1: aside from the one comment

**Comment by athal7** [comment]
lgtm. i assume this is related to https://github.com/bonobos/quiet_logistics_integration/pull/18#discussion_r30230715 ?

**Comment by philbirt** [comment]
cc @jordan-brough Updated spec to have a non-zero canceled qty test

**Comment by jordan-brough** [comment]
:+1:

**Comment by jordan-brough** [comment]
@athal7 somewhat related.  this is to allow us to transform the order when sending to simparel to subtract out the canceled units from the count.

**Pull request merged**